README.md

```diff
@@ -99,6 +99,10 @@ expect.extend({toBeInTheDOM, toHaveClass})
 
 ### `toBeInTheDOM`
 
+```
+toBeInTheDOM(container?: HTMLElement)
```

Comment:
Since #29 we also support `SGVElement`, which for some reason is a class of its own. I guess we missed reflecting this change in the typescript types too, so maybe we could take the opportunity to reflect the change in that file as well as in the README.

@jgoz since you worked in #29 and also have worked on TypeScript stuff in this repo, can you confirm the typescript signatures need to be updated to reflect that they support sag elements as well?

Reply:
@gnapse That's a good question. I suppose it's possible that someone would be using an SVG element as a container, so maybe we should update the signature as follows (both in the readme and in definitions):

```ts
toBeInTheDOM(container?: HTMLElement | SVGElement)
```

Reply:
Good to know. Are you up for it @jgoz? If not don't worry, maybe @smacpherson64 or I can do it later.

Reply:
For sure, I can add it to my list today.